homeassistant/components/deutsche_bahn/sensor.py

```diff
@@ -81,20 +86,22 @@ def update(self):
 class SchieneData:
     """Pull data from the bahn.de web page."""
 
-    def __init__(self, start, goal, only_direct):
+    def __init__(self, start, goal, time_delta, only_direct):
         """Initialize the sensor."""
         import schiene
 
         self.start = start
         self.goal = goal
+        self.time_delta = time_delta
         self.only_direct = only_direct
         self.schiene = schiene.Schiene()
         self.connections = [{}]
 
     def update(self):
         """Update the connection data."""
         self.connections = self.schiene.connections(
-            self.start, self.goal, dt_util.as_local(dt_util.utcnow()),
+            self.start, self.goal,
+            dt_util.as_local(dt_util.utcnow())+self.time_delta,
```

Comment:
```suggestion
            dt_util.as_local(dt_util.utcnow()+self.time_delta),
```

Otherwise you will get issues around daylight savings time switches.